punx/__init__.py

```diff
@@ -63,15 +61,17 @@
 __settings_organization__ = __package_name__
 __settings_package__ = __package_name__
 
-__install_requires__ = [
-    "h5py",
-    "lxml",
-    "numpy",
-    "pyRestTable",
-    "requests",
-    # 'PyQt5',
-    # see: http://stackoverflow.com/questions/4628519/is-it-possible-to-require-pyqt-from-setuptools-setup-py
-]
+__install_requires__ = """
+    h5py
+    hdf5plugin
```

Comment:
Technically not required, but as developer you can add to mandatory dependencies.